IllegalArgumentException: Required value was null

Sentry Url: https://sentry.io/share/issue/a32c45528c414af8973965f6c9edd293/
Senry Url 2: https://sentry.io/share/issue/fa68ef7d97294b979c94299bb1d78f0d/

```
java.lang.IllegalArgumentException: Required value was null.
    at org.wordpress.android.ui.posts.editor.StoriesEventListener.onStoryFrameSaveCompleted(StoriesEventListener.kt:123)
    at java.lang.reflect.Method.invoke(Method.java)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:507)
    at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:501)
    at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
    at android.os.Handler.dispatchMessage(Handler.java:111)
    at android.os.Looper.loop(Looper.java:194)
    at android.app.ActivityThread.main(ActivityThread.java:5648)
    at java.lang.reflect.Method.invoke(Method.java)
    at java.lang.reflect.Method.invoke(Method.java:372)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```


Logs for the user show a low memory situation, let's keep an eye on this and see if it has more occurences:

```
state: saveInstanceState
screen: PagesActivity
info
19:48:43
ui.lifecycle
state: stopped
screen: PagesActivity
info
19:48:43
generic
d/WordPress-API: ReactNativeStore: executeRequest
info
19:48:44
generic
d/WordPress-API: Dispatching action: PostAction-UPDATE_POST
info
19:48:46
generic
d/WordPress-API: Dispatching action: ListAction-LIST_DATA_INVALIDATED
info
19:48:46
generic
d/WordPress-POSTS: PageStore: getPagesFromDb
info
19:48:46
device.event
Low memory
level: 10
action: LOW_MEMORY
warning
19:49:03
exception
IllegalArgumentException: Required value was null.
```

I was able to reproduce this one on [`17.0-rc-2` ](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/17.0-rc-2)

```
    --------- beginning of crash
2021-03-25 13:34:44.813 24971-24971/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android, PID: 24971
    org.greenrobot.eventbus.EventBusException: Invoking subscriber failed
        at org.greenrobot.eventbus.EventBus.handleSubscriberException(EventBus.java:530)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:512)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:504)
        at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: java.lang.IllegalArgumentException: Required value was null.
        at org.wordpress.android.ui.posts.editor.StoriesEventListener.onStoryFrameSaveCompleted(StoriesEventListener.kt:173)
        at java.lang.reflect.Method.invoke(Native Method)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:510)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:504) 
        at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7356) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930) 
```
Steps to reproduce:
1. create a story and publish it
2. create a new Post
3. observe the crash

--- a/WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StoriesEventListener.kt
+++ b/WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StoriesEventListener.kt
@@ -166,10 +166,14 @@ class StoriesEventListener @Inject constructor(
     @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
     fun onStoryFrameSaveCompleted(event: FrameSaveCompleted) {
         eventBusWrapper.removeStickyEvent(event)
+        // in onStoryFrameSaveCompleted we should always get a frameId that has the TEMPORARY_ID_PREFIX prefix
+        // for Stories being edited only
+        // otherwise we can exit
+        if (event.frameId == null) return
+
         if (!lifecycle.currentState.isAtLeast(CREATED)) {
             return
         }
-        // in onStoryFrameSaveCompleted we should always get a frameId that has the TEMPORARY_ID_PREFIX prefix
         val localMediaId = requireNotNull(event.frameId)
 
         val (frames) = storyRepositoryWrapper.getStoryAtIndex(event.storyIndex)

--- a/WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StoriesEventListener.kt
+++ b/WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StoriesEventListener.kt
@@ -174,7 +174,7 @@ class StoriesEventListener @Inject constructor(
         if (!lifecycle.currentState.isAtLeast(CREATED)) {
             return
         }
-        val localMediaId = requireNotNull(event.frameId)
+        val localMediaId = event.frameId
 
         val (frames) = storyRepositoryWrapper.getStoryAtIndex(event.storyIndex)
 

